### ISSUE TITLE
Blitz Showcase

### ISSUE THREAD

**Issue by robdrosenberg** [blitz-js/blitzjs.com#36]
Add a showcase page or section to the site to show off apps and businesses using Blitz

**Comment by Uneetpatel7** [comment]
I would like to work on this

**Comment by robdrosenberg** [comment]
Hi @Uneetpatel7, We'd love to have your help on this. We have to wait until the design is finished before work should start on it though. Is that okay? I'll be sure to let you know once the design is ready.

**Comment by Uneetpatel7** [comment]
Yeah sure

**Comment by flybayer** [comment]
Closing in favor of #154

**Comment by flybayer** [comment]
@Uneetpatel7 do you still want to build this? I forgot about this issue and opened #154. We want to go ahead and add this page asap before the new design is done.
